### PR TITLE
feat(project-manager-review): add project manager review component

### PR DESCRIPTION
**Pull request by sijav**

### CODE REVIEW

File: src/shared/project-manager-review/ProjectManagerReview.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    >
+      <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+        <Typography variant="h6">{review.project.name}</Typography>
+      </ExpansionPanelSummary>
+      <ExpansionPanelDetails>
+        <div className={classes.wrapper}>
+          <div>
+            <Typography gutterBottom>{i18n._('Performance to initial expectation')}: </Typography>
+            <Box className={classes.wrapperBox}>
+              {review.rating ? <EvaluationOutput value={review.rating as Evaluation} /> : <p></p>}
```
Comment (Pymossy) [inline src/shared/project-manager-review/ProjectManagerReview.tsx]: I think null is better
```suggestion
              {review.rating ? <EvaluationOutput value={review.rating as Evaluation} /> : null}
```

File: src/shared/project-manager-review/ProjectManagerReview.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { storiesOf } from '@storybook/react';
+import graphql from 'babel-plugin-relay/macro';
+import React from 'react';
+import { useLazyLoadQuery } from 'react-relay/hooks';
+import { ProjectManagerReviewQuery } from './__generated__/ProjectManagerReviewQuery.graphql';
+import { themeDecorator } from 'src/stories/decorators/themeDecorator';
+import { ProjectManagerReview } from './ProjectManagerReview';
+import { relayDecorator } from 'src/stories/decorators';
+
+const query = graphql`
```
Comment (Pymossy) [inline src/shared/project-manager-review/ProjectManagerReview.stories.tsx]: I think using constant data is better than using graphql

File: src/shared/project-manager-review/ProjectManagerReview.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      classes={{ root: classes.expansionPanelRoot, expanded: classes.expansionPanelExpanded }}
+      defaultExpanded={true}
+      elevation={0}
+    >
+      <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+        <Typography variant="h6">{review.project.name}</Typography>
+      </ExpansionPanelSummary>
+      <ExpansionPanelDetails>
+        <div className={classes.wrapper}>
+          <div>
```
Comment (Pymossy) [inline src/shared/project-manager-review/ProjectManagerReview.tsx]: I think it is better to split this two div as component

File: src/shared/project-manager-review/ProjectManagerReview.stories.tsx (hunk 2 above)
Comment (sijav) [reply src/shared/project-manager-review/ProjectManagerReview.stories.tsx]: maybe in another pr

File: src/shared/project-manager-review/ProjectManagerReview.tsx (hunk 3 above)
Comment (sijav) [reply src/shared/project-manager-review/ProjectManagerReview.tsx]: Will use Grid component here thanks

File: src/shared/project-manager-review/ProjectManagerReview.tsx (hunk 1 above)
Comment (sijav) [reply src/shared/project-manager-review/ProjectManagerReview.tsx]: Will use typography variant body 1 here with non breaking space

File: src/shared/project-manager-review/ProjectManagerReview.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            <Typography gutterBottom>{i18n._('Performance to initial expectation')}: </Typography>
```
[the remark this answers is not in the document]
Comment (sijav) [reply src/shared/project-manager-review/ProjectManagerReview.tsx]: I think Performance to initial expectation is better here will check with UI/UX designer thanks

File: src/shared/project-manager-review/ProjectManagerReview.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    },
+  } as CSSProperties,
+  expansionPanelExpanded: {} as CSSProperties,
+  wrapper: {
+    width: '100%',
+    '& > *': {
+      margin: theme.spacing(0, 0, 2),
+    },
+  } as CSSProperties,
+  wrapperBox: {
```
[the remark this answers is not in the document]
Comment (sijav) [reply src/shared/project-manager-review/ProjectManagerReview.tsx]: Done thanks

File: src/shared/project-manager-review/ProjectManagerReview.tsx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+        <div className={classes.wrapper}>
+          <div>
+            <Typography gutterBottom>{i18n._('Performance to initial expectation')}: </Typography>
+            <Box className={classes.wrapperBox}>
+              {review.rating ? <EvaluationOutput value={review.rating as Evaluation} /> : <p></p>}
+            </Box>
+          </div>
+          <div>
+            <Typography gutterBottom>{i18n._('Accomplishments')}:</Typography>
+            <Box className={classes.wrapperBox}>
```
[the remark this answers is not in the document]
Comment (sijav) [reply src/shared/project-manager-review/ProjectManagerReview.tsx]: Used output border instead thanks

[the remark this answers is not in the document]
Comment (sijav) [reply src/shared/project-manager-review/ProjectManagerReview.tsx]: Done thanks

**Pull request merged**